File: 1º Semestre/Programação de Computadores/Dev C++ exercícios/Lista de exercício 03/10 - while.cpp
/* Elabore um algoritmo que solicite ao usuário uma palavra e um número inteiro que
indicará a quantidade de vezes que a palavra digitada será impressa na tela, um em cada
linha. (while) */

#include <stdio.h>

int main(){
	char palavra[30];
	int numero;
	int contador = 1;
	
	printf("Informe uma palavra: ");
	scanf("%s", &palavra);
	
	printf("Informe a quantidade de vezes que essa palavra sera impressa na tela: ");
	scanf("%d", &numero);
	
	while (contador <= numero){
		printf("%3d - %s\n", contador, palavra);
		contador++;
	}
	
	return 0;
	
}

File: 1º Semestre/Programação Python/Listas de Exercícios/Exercício 3/10 - while.py
'''10. Elabore um algoritmo que solicite ao usuário uma palavra e um número inteiro que
indicará a quantidade de vezes que a palavra digitada será impressa na tela, um em cada
linha.'''

contador = 1
palavra = str(input("Informe uma palavra: "))
numero = int(input("Informe um número inteiro: "))

while contador <= numero:
    print(f"{contador} - {palavra}")
    contador += 1

